scripts/measure_baseline.py

```
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+"""Run baseline benchmarks with CPU pinning and statistical aggregation across rounds."""
+
+import argparse
+import hashlib
+import os
+import platform
+from pathlib import Path
+import json
+import re
+import subprocess
+from typing import Dict, List, Any
```

Ruff (0.16.3)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)